standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/PartitionManagementTask.java

```diff
@@ -80,6 +82,16 @@ public Configuration getConf() {
     return conf;
   }
 
+  private static boolean partitionDiscoveryEnabled(Map<String, String> params) {
+    return params != null && params.containsKey(DISCOVER_PARTITIONS_TBLPROPERTY) &&
+            params.get(DISCOVER_PARTITIONS_TBLPROPERTY).equalsIgnoreCase("true");
+  }
+
+  private static boolean tblBeingReplicatedInto(Map<String, String> params) {
+    return params != null && params.containsKey(ReplConst.REPL_TARGET_TABLE_PROPERTY) &&
+            !params.get(ReplConst.REPL_TARGET_TABLE_PROPERTY).trim().isEmpty();
```

Comment:
isEmpty() check looks redundant

Reply:
params may contain REPL_TARGET_TABLE_PROPERTY but it may be an empty string ''. That will not happen through the code since we always add non-empty string. But this is an extra check to cover that possibility in case somebody adds an empty string there.